File: PYTHON/Pertemuan 01/Motor.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
from Kendaraan import Kendaraan

class Motor(Kendaraan):
    def __init__(self, merek: str):
        super().__init__(merek)

    def jalan(self):
        self.kecepatan = 90
        print(f"[Motor:{self.merek}] Ngebut {self.kecepatan} km/j!")

File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/Motor.java
package id.ac.uph.pbo.pertemuan01;

public class Motor extends Kendaraan {
    public Motor(String merek) { super(merek); }

    @Override
    public void jalan() {
        kecepatan = 90;
        System.out.println("[Motor:" + merek + "] Ngebut " + kecepatan + " km/j!");
    }
}
